templates/Commerce.Empty/.env

```diff
@@ -0,0 +1,5 @@
+# These variables are used for passing values into docker-compose.yml and the containers
+SA_PASSWORD=Qwerty12345!
+DB_NAME=Commerce.Empty.1_cms
+DB_NAME_COMMERCE=Commerce.Empty.1_commerce
+DB_DIRECTORY=commerce-empty
```

Comment:
```suggestion
DB_DIRECTORY=Commerce.Empty.1
```

Reply:
@JohanPetersson Fixed in [5d0b43e](https://github.com/episerver/content-templates/pull/47/commits/5d0b43ee2a5dcd7bccd2e4e3ffb0a5398688f3fd)